sharding-transaction/src/main/java/io/shardingsphere/transaction/manager/xa/XATransactionManager.java

```diff
@@ -30,6 +30,17 @@
  * @author zhangliang
  */
 public interface XATransactionManager extends ShardingTransactionManager<XATransactionEvent> {
+
+    /**
+     * initialize the transaction manager
+     */
+    void init() throws Exception;
```

Comment:
I think we can remove this init() method, and do initialize in constructor of AtomikosTransactionManager directly

Reply:
that makes sense.